### DISCUSSION TITLE
Brush size not sync/follow the canvas size

### DISCUSSION THREAD

**Discussion by Mnsayob** [keiwando/toonsquid#273] [General]
Brush size change when i zoom the canvas. The brush follow your screen size and not the canvas size. When i draw dot on zoomed out canvas, the dot will be bigger and thic

**Comment by Cat-Butler** [comment]
If this is implemented, please make it optional.
There are times when I want to zoom in to edit finer detail with a brush proportionate to the zoom level, and other times when I want to zoom in (to e.g. close a gap or edit a tricky area) and still use the same brush size proportionate to the canvas size (so it doesn't look weird when I zoom back out again).

**Comment by keiwando** [comment]
There was a bug that could cause unexpected vector brush sizes after zooming (which was more noticeable if the brush had a low maximum size setting).

This should now be fixed with the 1.8.5 update.
